webserver/templates/base.html

```
@@ -27,6 +27,31 @@
 
     <script type="text/javascript" src="{{ url_for('.static', filename='js/webserver.js') }}"></script>
 
+    <script type="text/javascript">
+        // Search for all "lang" tags and translate the content if possible.
+        $(document).ready(function() {
+            $.getJSON("{{ url_for('.static', filename='language/' + browser_language + '.json') }}", function(jsonData){
```

Adding an error handler for the `getJSON` request would be nice.

---

For all non German translation, the error handler will be called. What should the error handler do?

---

I meant an error handler for the case when the JSON file could not be retrieved.
It would be nice to display an error message in this case, too.
So we do not silently ignore an error.

---

In this case, on all non german accesses the user will see an error message. That is not really usable.